iml-gui/crate/src/page/partial/header.rs

```diff
@@ -193,6 +186,23 @@ fn main_menu_items(model: &Model) -> Node<Msg> {
     ]
 }
 
+fn context_sensitive_help_link(model: &Model, menu_class: &Attrs) -> Node<Msg> {
+    let attrs = if let Some(help_link) = model.route.help_link() {
+        // open the link in a new tab
+        attrs! { At::Target => "_blank", At::Href => help_link }
+    } else {
+        attrs! { At::Href => "" }
```

Comment:
If we happen to hit a page that doesn't have a corresponding help page, we should default to just opening `CTX_HELP` I think. The `help_link` could just return `Option<String>` where `String` is `"#<SPECIFIC_SECTION_HERE"`. Then this code could look something like:

 ```rust
    let attrs = attrs! {
        At::Target => "_blank",
        At::Href => format!(
            "{}{}",
            &CTX_HELP,
            model.route.help_link().unwrap_or_else(|| "".into())
        )
    };
```